intro_to_python/intro_to_python.md

```diff
@@ -2,21 +2,22 @@
 
 author:   Meredith Lee
 email:    leemc@chop.edu
-version:  1.0.0
+version:  2.0.0
```

Comment:
```suggestion
version:  1.0.0
```
since we're now entirely renaming + relocating the module, do we think we should go back to a 1.0.0 version?

Reply:
For a mom,ent I thought this may have been covered in the versioning guidelines, but I just looked back and it doesn't look like it was (specifically how to handle when the rescoping involves renaming). I'm leaning toward doing 1.0.0 for this, but haven't thought on it particularly hard so feel free to push back!